plugins/system/mvcoverride/language/en-GB/en-GB.plg_system_mvcoverride.ini

```diff
@@ -4,3 +4,9 @@ PLG_SYSTEM_MVC_OVERRIDE_MAKE_EXTENDABLE="Make Extendable"
 PLG_SYSTEM_MVC_OVERRIDE_MAKE_EXTENDABLE_DESC="Extend core component class. Its better to use OOP to extend components. You will be able to extend your override classes extend a Component[Controller|Model|View][Name]Default"
 PLG_SYSTEM_MVC_OVERRIDE_CHANGE_FUNCTIONS="Change functions"
 PLG_SYSTEM_MVC_OVERRIDE_CHANGE_FUNCTIONS_DESC="Changes all private methods in the base class to protected methods which allows the subclass to extend the base class better. This is not recommended unless you really need it."
+PLG_SYSTEM_MVC_OVERRIDE_PREFIX_LABEL="Prefix for extends classes"
+PLG_SYSTEM_MVC_OVERRIDE_PREFIX_DESC="Set prefix for exten classes. <br /><strong>Warning: at least one value must be set prefix or suffix or both</strong>"
+PLG_SYSTEM_MVC_OVERRIDE_SUFFIX_LABEL="Suffix for extend classes"
+PLG_SYSTEM_MVC_OVERRIDE_SUFFIX_DESC="Set suffix for extend classes. <br /><strong>Warning: at least one value must be set prefix or suffix or both</strong>"
+PLG_SYSTEM_MVC_OVERRIDE_INCLUDE_PATH_LABEL="Include pathes"
```

Comment:
same in the following line, use "paths" instead of "pathes"

```
PLG_SYSTEM_MVC_OVERRIDE_INCLUDE_PATH_DESC="list of include paths. Use comma as separator. <br />Use the tag {JPATH_BASE} to set JPATH_BASE constant in path. <br />Use the tag {JPATH_THEMES}  to set  JPATH_THEMES in path. <br />Use tag {template} to set include your template folder. <br />Example of usage: {JPATH_BASE}/code, {JPATH_THEMES}/{template}/code. The last path of the list has the first priority when loading."
```